_emails/us/california/san_francisco.md

```diff
@@ -30,7 +30,7 @@ recipients:
 - JennyLam@sfusd.edu
 - FaauugaMoliga@sfusd.edu
 - RachelNorton@sfusd.edu
-body: |-
```

Comment:
Why was this removed? It will break this markdown file. Pipe character is needed, or newlines must be explicitly state with \n characters - pipe is the preferred pattern in our case.

Reply:
accident? reverted